### PR TITLE
minor changes: MANIFEST.in to include docs, manifold doc update, minor fix to manifold data_frame input

### PR DESCRIPTION
**Pull request by kenns29**

### CODE REVIEW

File: bindings/jupyter/docs/manifold.md [hunk 1]
```diff
@@ -53,15 +54,15 @@ data = {
   x:     [...],         # feature data
   yPred: [[...], ...]   # prediction data
   yTrue: [...],         # ground truth data
-}
+};
```
Comment (ajduberstein) [inline bindings/jupyter/docs/manifold.md]: The code block is python, so you likely want no semi-colons

File: bindings/jupyter/docs/manifold.md [hunk 2]
```diff
@@ -53,15 +54,15 @@ data = {
...
 ```
 
 Each element in these lists represents one data point in your evaluation dataset, and the order of data instances in `x`, `yPred` and `yTrue` should all match.
 Recommended instance count for each of these datasets is 10000 - 15000. If you have a larger dataset that you want to analyze, a random subset of your data generally suffices to reveal the important patterns in it.
 
-##### `x` (list | ndarray, required):
+##### `x` (list | ndarray | data_frame, required):
+A list/ndarray/data_frame of instances with features. Example (2 data instances):
```
Comment (ajduberstein) [inline bindings/jupyter/docs/manifold.md]: Nit: Use the library + type name, like `pandas.DataFrame` and `numpy.ndarray`

Comment (kenns29) [reply bindings/jupyter/docs/manifold.md]: addressed.

File: bindings/jupyter/docs/manifold.md (hunk 1 above)
Comment (kenns29) [reply bindings/jupyter/docs/manifold.md]: addressed.

**Pull request merged**